satyr.spec.in

```diff
@@ -1,26 +1,7 @@
-%{!?python2_sitearch: %global python2_sitearch %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print get_python_lib(1)")}
-
-%if 0%{?fedora} || 0%{?rhel} > 7
-# Enable python3 build by default
```

Comment:
I think this should stay. We want to build `python3-satyr` by default.

Reply:
https://github.com/abrt/satyr/blob/b1631124e0571efc6baa1383192d7e41dee4122c/configure.ac#L64

Looks to me like it’s enabled by default anyway.

Reply:
https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/autoconf.html#External-Software

`action-if-not-given` sets `with_python3` to `yes`, so the option is just for disabling it.

Reply:
https://rpm.org/user_doc/conditional_builds.html

It defaults to `--without-python3`.

Reply:
Right, seems that we’re talking about completely different things here.

Reply:
In this case, it wouldn’t default to anything, no? I removed the %bcond_with{,out} lines.

Reply:
In that case, I should probably have also removed the conditional when executing `configure`. Do you think it makes sense to keep the conditionals?

Reply:
Keep the `%bcond_without python3` in this PR. We can remove the with/without python3 conditions in the future.